File: app/src/main/java/com/something15525/trimetgo/trimet_go/data/model/Arrival.java
package com.something15525.trimetgo.trimet_go.data.model;

import java.util.ArrayList;
import org.joda.time.DateTime;

public class Arrival {

    private String fullSign;
    private String shortSign;
    private DateTime estimated;
    private DateTime scheduled;
    private ArrayList<Detour> detours;
    private int route;
    private String status;

    public ArrayList<Detour> getDetours() {
        return this.detours;
    }

    public DateTime getEstimated() {
        return this.estimated;
    }

    public String getFullSign() {
        return this.fullSign;
    }

    public int getRouteId() {
        return this.route;
    }

    public DateTime getScheduled() {
        return this.scheduled;
    }

    public String getShortSign() {
        return this.shortSign;
    }

    public String getStatus() {
        return this.status;
    }

    public void setFullSign(String fullSign) {
        this.fullSign = fullSign;
    }

    public void setShortSign(String shortSign) {
        this.shortSign = shortSign;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public void setEstimated(DateTime estimated) {
        this.estimated = estimated;
    }

    public void setScheduled(DateTime scheduled) {
        this.scheduled = scheduled;
    }

    public void setRouteId(int route) {
        this.route = route;
    }

    public void setDetours(ArrayList<Detour> detours) {
        this.detours = detours;
    }
}

File: app/src/main/java/com/trimettransit/tracker/data/model/Arrival.kt
package com.trimettransit.tracker.data.model

import org.joda.time.DateTime

data class Arrival(
    var fullSign: String = "",
    var shortSign: String = "",
    var estimated: DateTime? = null,
    var scheduled: DateTime? = null,
    var detours: MutableList<Detour>? = null,
    var routeId: Int = 0,
    var status: String = "",
    var dropOffOnly: Boolean = false,
    var reason: String = "",
    var tripID: String = "",
    var blockID: Int = 0,
    var vehicleID: Int = 0,
    var feet: Int = 0,
    var dir: Int = 0,
    var estimatedMillis: Long = 0,
    var scheduledMillis: Long = 0
)
